examples/grafana/README.md

```diff
@@ -21,20 +22,22 @@ pyrra_errors_total
 ```
 
 The Grafana dashboards rely on these recording rules. You need to enabled generating these on the `filesystem` or `kubernetes` component.
-Run these components with the `--generic-rules` flag. 
+Run these components with the `--generic-rules` flag.
 
-Please note that the `grouping` feature is not supported when using generic rules. 
+Please note that the `grouping` feature is not supported when using generic rules.
 This is a limitation with the Grafana dashboard dropdowns (they would need to be dynamic).
 
+You will need at least one rule without `grouping` to generate the `pyrra_` metrics that are required to make the Pyrra dashboards work.

```

Comment:
That's a good one! Thanks for adding that.
As I'm working on validation now, do you think if the generic rules are enabled we should entirely fail validation, or is a warning enough?

Reply:
This depends on how you handle this for the project, and what you think is best for Pyrra's users.

If you want to forbid the use of grouping when using genericRules, then yes, I would expect the validation to fail.

If it makes sense to keep it the way it is, I think the documentation will already help most users. Maybe a warning log could be a nice addition too.

Reply:
Yeah, the warning is probably the best option. 
Given that people might run Pyrra with its own UI but also display some SLOs within Grafana dashboards, failing is probably the wrong choice. 

Thanks for your input!